### PR TITLE
add elinga_seinauskaite.html file

### PR DESCRIPTION
**Pull request by seinauskaite**
created html and added Hey

### CODE REVIEW

**Review by MarcinkEve — APPROVED** [review]

**Review by MartynasWoo — APPROVED** [review]

File: elinga_seinauskaite.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hey</h1>
```
Comment (suiraDN) [inline elinga_seinauskaite.html:10]: Hey

**Review by suiraDN — APPROVED** [review]

**Review by SwallowEve — APPROVED** [review]
Hey-hey :)

File: elinga_seinauskaite.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (eciras) [inline elinga_seinauskaite.html:7]: Change title name.

**Review by eciras — APPROVED** [review]

**Review by MartynasKupsys — APPROVED** [review]
Vuolia!!

**Review by MrStonkus — APPROVED** [review]

**Pull request merged**